### PR TITLE
Update sc hero

### PR DESCRIPTION
**Pull request by Gup-B**
Added in Enddate option to macro
Removed signpost link for hero
Updated css to use new vars

### CODE REVIEW

File: packages/shared-component--hero/src/hero.pcss [hunk 1]
```diff
@@ -5,268 +5,200 @@
...
-    clear: both;
-    display: table;
+}
+@media (min-width: 81.25em) {
+  .coop-c-hero__inner {
+    padding: var(--spacing-48) var(--spacing-16) 0;
   }
 }
-
+.coop-c-hero__inner:after {
```
Comment (colinrotherham) [inline packages/shared-component--hero/src/hero.pcss:46]: Do you want to use `.coop-u-clearfix` instead?

File: packages/shared-component--hero/src/hero.pcss [hunk 2]
```diff
@@ -5,268 +5,200 @@
...
   display: inline-block;
-  padding: var(--spacing-base--1-4) calc(var(--spacing-base--1-2) * 1.25);
+  padding: 0.5rem 1.25rem;
   color: var(--color-white);
-  background: var(--color-button-primary);
-  font-size: 1.125rem;
-  font-family: var(--font-family);
+  background: var(--color-button-green-primary);
+  font-size: var(--type-sp-l);
+  font-family: Avenir-Next, Helvetica Neue, Helvetica, Arial, sans-serif;
```
Comment (colinrotherham) [inline packages/shared-component--hero/src/hero.pcss:157]: Can this entire block be ripped out and swapped with a `coop-btn coop-btn--primary` class in the HTML?

File: packages/shared-component--hero/src/hero.pcss [hunk 3]
```diff
@@ -5,268 +5,200 @@
...
   transition: all 0.15s linear;
   outline: none;
   text-decoration: none;
-
-  @media (--mq-large) {
-    font-size: 1.25rem;
+}
+@media (min-width: 64em) {
+  .coop-c-hero__cta__link {
+    ont-size: var(--type-sp-l);
```
Comment (colinrotherham) [inline packages/shared-component--hero/src/hero.pcss:168]: Missing `f` on font

Comment (colinrotherham) [reply packages/shared-component--hero/src/hero.pcss:168]: But probably can be swapped with `coop-btn` in the HTML

File: packages/shared-component--hero/src/hero.pcss (hunk 2 above)
Comment (Gup-B) [reply packages/shared-component--hero/src/hero.pcss]: css removed

File: packages/shared-component--hero/src/hero.pcss (hunk 3 above)
Comment (Gup-B) [reply packages/shared-component--hero/src/hero.pcss]: css removed

File: packages/shared-component--hero/src/hero.pcss [hunk 4]
```diff
@@ -275,40 +165,40 @@
   transform: translateX(-50%);
   margin-left: -10%;
   z-index: 1;
-  fill: #f3f9fd;
+  fill: #f2f4f7;
```
Comment (colinrotherham) [inline packages/shared-component--hero/src/hero.pcss:168]: Mind if we use `var(--color-grey-neutral-cool-light)`?

**Pull request merged**